File: 1480.running-sum-of-1-d-array.java
/*
 * @lc app=leetcode id=1480 lang=java
 *
 * [1480] Running Sum of 1d Array
 */

// @lc code=start
class Solution {
    public int[] runningSum(int[] nums) {

        int[] output = new int[nums.length];


        for (int i =0;i<nums.length;i++){
            if (i==0){
                output[i]=nums[i];
            }
            else{
                output[i]=output[i-1] + nums[i];
            }
        }

        return output;
    }
}
// @lc code=end

File: 1480_Running Sum of 1d Array.py
class Solution(object):
    def runningSum(self, nums):
        output=[]
        a=0
        for i in nums:
            a = a+i
            output.append(a)
        return output
        """
        :type nums: List[int]
        :rtype: List[int]
        """
"""
Runtime: 32 ms, faster than 59.31% of Python online submissions for Running Sum of 1d Array.
Memory Usage: 12.8 MB, less than 100.00% of Python online submissions for Running Sum of 1d Array.
"""
